src/main/kotlin/uk/gov/dluhc/printapi/messaging/service/AbstractEmailSender.kt

```diff
@@ -0,0 +1,30 @@
+package uk.gov.dluhc.printapi.messaging.service
+
+import uk.gov.dluhc.printapi.client.ElectoralRegistrationOfficeManagementApiClient
+import uk.gov.dluhc.printapi.database.entity.Certificate
+import uk.gov.dluhc.printapi.database.entity.PrintRequest
+import uk.gov.dluhc.printapi.messaging.models.ProcessPrintResponseMessage
+
+abstract class AbstractEmailSender(
+    private val electoralRegistrationOfficeManagementApiClient: ElectoralRegistrationOfficeManagementApiClient,
```

Comment:
Can you not use `EroService` instead?

Reply:
Not with adding a new method that would just invoke the electoralRegistrationOfficeManagementApiClient